tests/std/tests/P0896R4_ranges_alg_remove_if/test.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <ranges>
+#include <span>
+#include <utility>
+
+#include <range_algorithm_support.hpp>
+using namespace std;
+using P = pair<int, int>;
+
+constexpr auto matches = [](int const val) { return val == 47; };

```

Comment:
Nitpick: This is using east `const`, but the lambda below (and the other files) are using west `const`.

Reply:
Done